rust/signed_doc/tests/decoding.rs

```diff
@@ -39,7 +39,7 @@ fn catalyst_signed_doc_cbor_roundtrip_test() {
 fn catalyst_signed_doc_cbor_roundtrip_kid_as_id_test() {
     let (_, _, metadata_fields) = common::test_metadata();
     let (sk, _, kid) = create_dummy_key_pair(RoleIndex::ROLE_0).unwrap();
-    // transform Catalyst ID URI form to the ID form
+    // transform Catalyst ID form to the ID form
```

Comment:
No no, revert this comment.
This `CatalysId` structure is a generic abstraction over Catalys Identifier object which could be both in the URI form or only in the so-called "short"/"ID" form